monitoring/publishing/db_update_sqlite.py

```diff
@@ -0,0 +1,302 @@
+# -*- coding: utf-8 -*-
+"""
+`db_update_sqlite.py` - Syncs data from external database into local SQLite DB.
+                      -  It will be run as a standalone operation via cron.
+"""
+import configparser
+import logging
+import os
+import sys
+
+import django
+from django.db import DatabaseError
+import pandas as pd
+
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+# Find the root and the Django project
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..')))
```

Comment:
Aren't these two paths equivalent? Wouldn't it be neater to do something like this?

```suggestion
BASE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))

sys.path.append(BASE_DIR)
```

Reply:
Sure. Done